src/commands/math/commands.js

```diff
@@ -369,11 +373,11 @@ var SummationNotation = P(MathCommand, function(_, super_) {
   };
 });
 
-LatexCmds['∑'] =
+LatexCmds['?'] =
 LatexCmds.sum =
 LatexCmds.summation = bind(SummationNotation,'\\sum ','&sum;');
 
-LatexCmds['∏'] =
+LatexCmds['?'] =
```

Comment:
Looks like this happened in https://github.com/sclower/mathquill/commit/f53cfb10caacbc1f9bf3b1d31f5589cabb78a199. It seems like a mistake to me.